context/app/static/js/components/detailPage/visualization/Visualization/Visualization.tsx

```diff
@@ -68,6 +68,17 @@ function Visualization({
   useCanvasScrollFix();
   const { toastError, toastInfo } = useSnackbarActions();
 
+  const handleWarning = useCallback(
+    (message: string) => {
+      // Suppress the "Node not found" message that appears when no zarr file was found
+      if (message.includes('Node not found')) {
```

Comment:
I'm seeing a `Unexpected response status 401` warning from what appears to be the same zarr issue, based on the network logs: 
![image](https://github.com/user-attachments/assets/b3ee99c9-4549-47e2-a7a2-16fdfb1a0241)
![image](https://github.com/user-attachments/assets/4bfa3d25-d0a0-4d06-939f-beb5ed92a2fc)

I think we could add that as another message to match against.